.gitignore

```diff
@@ -45,7 +47,11 @@ yarn-error.log
 buck-out/
 \.buckd/
 android/app/libs
-android/keystores/debug.keystore
+*.keystore
+!debug.keystore
```

Comment:
im not sure about this one

Reply:
Was added by RN official migration guide.